### DISCUSSION TITLE
Configure scatter chart?

### DISCUSSION THREAD

**Discussion by jacksonmacd** [team-moeller/better-access-charts#109] [Q&A]
How do I designate which fields to use for the X and Y axis of a scatter chart? Seems like the Y is fairly obvious (.DataSource.DataFieldNames) but I can't figure out where to designate the field to use for the X-axis.

Thanks in advance

**Comment by team-moeller** [comment]
The first "DataFieldName" in the data source is for the x-axis, the second one ist for the y-axis.
